.changeset/calm-horses-feel.md

```diff
@@ -0,0 +1,11 @@
+---
+"undefined": minor
```

Comment:
Is this "undefined" package intentional/necessary? I haven't seen this before, so I'm curious.

Reply:
To add a changeset, I initially ran `pnpm changeset` from the root, which created the changeset file. After reviewing the documentation, it seems the correct command to use is `pnpm flub changeset add`. updating this